src/Storage/StorageClient.php

```diff
@@ -217,4 +223,53 @@ public function createBucket($name, array $options = [])
         $response = $this->connection->insertBucket($options + ['name' => $name, 'project' => $this->projectId]);
         return new Bucket($this->connection, $name, $response);
     }
+
+    /**
+     * Register a StreamWrapper for reading and writing to Google Storage
+     *
+     * @param string $protocol The name of the protocol to use. Defaults to
+     *        'gs'.
+     * @throws \RuntimeException
+     */
+    public static function registerStreamWrapper(string $protocol = null)
```

Comment:
This was more along the lines of what people may expect when they normally register a stream wrapper - just with the handler class being taken care of for them. Changing it do an method on a `StorageClient` also makes sense.